tcf_website/templates/course/course.html

```diff
@@ -26,17 +26,38 @@ <h1 class="mr-md-3 text-nowrap">{{ course.code }}</h1>
                 <h2>{{ course.title }}</h2>
             </div>
 
-            <a data-toggle="tooltip"
-               data-placement="right"
-               title="Requires NetBadge Login"
-               class="text-tcf-blue font-weight-bold"
-               target="_blank"
-               href={{ course.eval_link }}>
-                Official UVA Student Evaluations
-                <i class="fa fa-external-link" aria-hidden="true"></i>
-            </a>
+            <!-- Course Description -->
+            {% if course.description|length > 0 %}
+            <div class="card-body">
```

Comment:
Not too sure on this one, but you might have to wrap this in a `class="card"` div to get the card background to show up properly.